python/tools/auditwheel

```diff
@@ -0,0 +1,13 @@
+#!/opt/_internal/pipx/venvs/auditwheel/bin/python
+#
+# Monkey patch to not ship libarrow(*).so in pypi wheels
+import sys
+
+from auditwheel.main import main
+from auditwheel.policy import _POLICIES as POLICIES
+
+for p in POLICIES:
+    p['lib_whitelist'].extend(['libarrow.so.800', 'libarrow_dataset.so.800'])
```

Comment:
Does pyarrow also ship stuff like libparquet.so?

Reply:
yes, `pyarrow` ships it, and lance lib does not use `libparquet`, we only use it in benchmark.